ui/src/treeDataProvider/AzExtTreeDataProvider.ts

```diff
@@ -109,28 +111,42 @@ export class AzExtTreeDataProvider implements IAzExtTreeDataProviderInternal, ty
         this.refreshUIOnly(treeItem);
     }
 
-    public refreshUIOnly(treeItem: AzExtTreeItem | undefined): void {
-        this._onDidChangeTreeDataEmitter.fire(treeItem === this._rootTreeItem ? undefined : treeItem);
+    public refreshUIOnly(_treeItem: AzExtTreeItem | undefined): void {
+        // Pass undefined as temporary workaround for https://github.com/microsoft/vscode/issues/71698
+        this._onDidChangeTreeDataEmitter.fire(undefined);
```

Comment:
@nturinski this was your recommended workaround for "Element with id is already registered", right? I was getting that error all the time in Docker and couldn't handle it anymore. Plus the VS Code folks bumped the milestone on that issue, so I'm not sure when they'll get to it.

Anyways - just wanted to say thanks for finding the workaround!